.circleci/config.yml

```diff
@@ -24,20 +22,127 @@ jobs:
             SCHEME: Auth0.iOS
             DEVICE: iPhone 8
             FASTLANE_EXPLICIT_OPEN_SIMULATOR: 2
-      - run:
-          name: Pod Lib Lint
-          command: bundle exec fastlane ios pod_lint
       - run: |
-          bash <(curl -s https://codecov.io/bash) -J 'Auth0' 
+          bash <(curl -s https://codecov.io/bash) -J 'Auth0'
```

Comment:
Should we also upload the coverage for the rest of the build variants? Why do you need to pass the package name?

Reply:
I've ran into issues in the past with multiple codecov, I don't think it understands what to do with it for example. MacOS doesn't have WebAuth so if that report is sent after the iOS one codecov will report lots of missing codecov.